bin/installer.sh

```diff
@@ -1,9 +1,2 @@
 
-pyinstaller \
- --name="FreeMoCap" \
- --windowed \
- --onefile \
- --hidden-import pyqtgraph.graphicsItems.ViewBox.axisCtrlTemplate_pyqt6 \
- --hidden-import pyqtgraph.graphicsItems.PlotItem.plotConfigTemplate_pyqt6 \
- --hidden-import pyqtgraph.imageview.ImageViewTemplate_pyqt6 \
- ./src/gui/main/main.py
+pyinstaller  --name="FreeMoCap"  --onefile  --hidden-import pyqtgraph.graphicsItems.ViewBox.axisCtrlTemplate_pyqt6 --hidden-import pyqtgraph.graphicsItems.PlotItem.plotConfigTemplate_pyqt6 --hidden-import pyqtgraph.imageview.ImageViewTemplate_pyqt6  ./src/gui/main/main.py
```

Comment:
Make a new file in the Windows folder instead of changing the linux/macos install command.